common/src/JetCorrections.cxx

```diff
@@ -279,18 +371,27 @@ std::unique_ptr<FactorizedJetCorrector> build_corrector(const std::vector<std::s
       if(jet.v4().Pt() > 15 && (jet.neutralEmEnergyFraction()+jet.chargedEmEnergyFraction())<0.9){
 	auto factor_raw = jet.JEC_factor_raw();
 	auto L1factor_raw = jet.JEC_L1factor_raw();
+	auto factor_raw_JER = jet.JER_factor_raw();
 
-	corrector_L1RC.setJetPt(jet.pt() * factor_raw);
+	corrector_L1RC.setJetPt(jet.pt() * factor_raw_JER);
```

Comment:
setJetPt should take raw jet pt. This is not raw, but L123 corrected, right?

Reply:
and for corrector.setJetPt(jet.pt() * factor_raw); it should be jet.pt() * factor_raw * factor_raw_JER